qutebrowser/misc/elf.py

```diff
@@ -316,18 +316,10 @@ def parse_webenginecore() -> Optional[Versions]:
     else:
         library_path = pathlib.Path(QLibraryInfo.location(QLibraryInfo.LibrariesPath))
 
-    filenames = [
-        # PyQt bundles those files with a .5 suffix, and Linux distributions
-        # most likely symlinks that.
-        'libQt5WebEngineCore.so.5',
-        # OpenBSD
-        'libQt5WebEngineCore.so.1.0',
-    ]
-    for filename in filenames:
-        lib_file = library_path / filename
-        if lib_file.exists():
-            log.misc.debug(f"QtWebEngine .so found at {lib_file}")
-            break
+    library_name = sorted(list(pathlib.Path(library_path).glob('libQt5WebEngineCore.so*')))
+    lib_file = library_path / library_name[-1].as_posix()
```

Comment:
This will break if no such file was found (like on macOS or Windows). Also, the `.exists()` check below doesn't make much sense either - if it was found by `.glob()`, it should always exist.

Finally, why the `.as_posix()` here?